db/migrate/20230502022612_create_topics.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTopics < ActiveRecord::Migration[7.0]
+  def change
+    create_table :topics do |t|
+      t.string :name
+      t.string :image
```

Comment:
you added presence validations, so maybe you want to add null: false here too? so you add the restriction at database level also